src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -29,15 +29,13 @@ export type ClickableStyleProps<IComponent extends React.ElementType> = {
    */
   size?: 'sm' | 'md' | 'lg';
   /**
-   * Available _stylistic_ variations available for the ClickableStyle component
+   * Available _color_ variations available for the ClickableStyle component
```

Comment:
I changed the comments to say "color" and "shape" because they're both stylistic variations. Lemme know if you want it changed back.